skops/cli/_convert.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+import pathlib
+import pickle
+from typing import Optional
+
+from skops.cli._utils import get_log_level
+from skops.io import dumps, get_untrusted_types
+
+
+def _convert_file(
+    input_file: os.PathLike,
+    output_file: os.PathLike,
+    logger: logging.Logger = logging.getLogger(),
+) -> None:
+    """Function that is called by ``skops convert`` entrypoint.
+
+    Loads a pickle model from the input path, converts to skops format, and saves to
+    output file.
+
+    Parameters
+    ----------
+    input_file : os.PathLike
+        Path of input .pkl model to load.
+
+    output_file : os.PathLike
+        Path to save .skops model to.
+
+    """
+    model_name = pathlib.Path(input_file).stem
+
+    logger.debug(f"Converting {model_name}")
+
+    with open(input_file, "rb") as f:
+        obj = pickle.load(f)
+    skops_dump = dumps(obj)
+
+    untrusted_types = get_untrusted_types(data=skops_dump)
+
+    if not untrusted_types:
+        logger.info(f"No unknown types found in {model_name}.")
+    else:
+        untrusted_str = ", ".join(untrusted_types)
+
+        logger.warning(
+            f"While converting {model_name}, "
```

Comment:
```suggestion
            f"While converting {input_file}, "
```

Arguably this is a bit more precise?